spec/Api/ProductDraftUuidApiSpec.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace spec\Akeneo\Pim\ApiClient\Api;
+
+use Akeneo\Pim\ApiClient\Api\Operation\GettableResourceInterface;
+use Akeneo\Pim\ApiClient\Api\ProductDraftUuidApi;
+use Akeneo\Pim\ApiClient\Api\ProductDraftUuidApiInterface;
+use Akeneo\Pim\ApiClient\Client\ResourceClientInterface;
+use Akeneo\Pim\ApiClient\Pagination\PageFactoryInterface;
+use Akeneo\Pim\ApiClient\Pagination\ResourceCursorFactoryInterface;
+use PhpSpec\ObjectBehavior;
+
+class ProductDraftUuidApiSpec extends ObjectBehavior
+{
+    function let(
+        ResourceClientInterface $resourceClient,
+        PageFactoryInterface $pageFactory,
+        ResourceCursorFactoryInterface $cursorFactory
+    ) {
+        $this->beConstructedWith($resourceClient, $pageFactory, $cursorFactory);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType(ProductDraftUuidApi::class);
+        $this->shouldImplement(ProductDraftUuidApiInterface::class);
+        $this->shouldImplement(GettableResourceInterface::class);
+    }
+
+    function it_gets_a_product_draft($resourceClient)
+    {
+        $draft = [
+            'uuid' => '944ca210-d8e0-4c57-9529-741e17e95c8d',
```

Comment:
I think you still should return an identifier
`'identifier' => 'foo',`

Reply:
discussed: api spec doesn't mention `identifier` property